src/components/InputBar/InputBar.jsx

```diff
@@ -1,26 +1,12 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import './InputBar.css';
+import React from 'react';
```

Comment:
nit: I have pet peeve about spacing local and non-local (node_modules) imports apart by one line.